cmd/ethereum_rust/ethereum_rust.rs

```diff
@@ -45,6 +40,13 @@ async fn main() {
         return;
     }
 
+    let log_level = matches
+        .get_one::<String>("log-level")
+        .expect("shouldn't happen, log-level is used with a default value");
+    let log_level = Level::from_str(log_level).unwrap_or(Level::INFO);
```

Comment:
maybe we should fail if the given value for the log-level is not correct instead?
this is parsed when the node starts, right?

Reply:
Yes this is when the node starts, thought that as logs aren't really needed for the execution, we could have a default value.
Changing it to an `expect` with some error description

Reply:
Ok, I agree with that approach.